### DISCUSSION TITLE
Welcome to open.mp web services discussions!

### DISCUSSION THREAD

**Discussion by Southclaws** [openmultiplayer/web#77] [General]
## 👋 Welcome!

So it looks like our repo got access to GitHub Discussions! Nice!

Use this to discuss open.mp web things, the site, the server list, the API, the forum, how game servers will interact with this. Share ideas amongst community members, contributors and team members!

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kaeygta** [comment]
Hello! I’m Potassium, I’ve been around the SA-MP community for about 10 years 😍 I am here as the open.mp community manager to help keep SA-MP alive and bring the community back together 🥰 One day I’ll learn to code and be able to contribute to that side of the project too 😉 (look out for our “Potassium learns Pawn” streams!!!)
